lib/checkMaster.js

```diff
@@ -1,69 +0,0 @@
-/*
```

Comment:
This PR changes de way HA works. Thus, I understand that architecture.md should be modified to describe the new mechanism and adapt pictures accordingly.